service/e2e_tests/base.py

```diff
@@ -0,0 +1,108 @@
+        # There seems to be a running query for the filter tree which doesn't terminate in time.
+        with connection.cursor() as c:
+            c.execute("SELECT pg_terminate_backend(pg_stat_activity.pid) FROM pg_stat_activity WHERE pg_stat_activity.datname = 'test_edushare' AND pid <> pg_backend_pid();")
+
+@override_settings(ELASTICSEARCH_NL_INDEX="test-nl", ELASTICSEARCH_EN_INDEX="test-en")
+class ElasticSearchTestCase(BaseTestCase):
+    @classmethod
+    def index_body(cls, language):
+        # TODO: Share config with harvester somehow
+        analyzer = 'dutch' if language == 'nl' else 'english'
+        return {
+            "mappings": {
+                "_doc": {
+                    "properties": {
+                        'title': {
+                            'type': 'text',
+                            'analyzer': analyzer
+                        },
+                        'text': {
+                            'type': 'text',
+                            'analyzer': analyzer
+                        },
+                        'transcription': {
+                            'type': 'text',
+                            'analyzer': analyzer
+                        },
+                        'description': {
+                            'type': 'text',
+                            'analyzer': analyzer
+                        },
+                        'url': {'type': 'text'},
+                        'title_plain': {'type': 'text'},
+                        'text_plain': {'type': 'text'},
+                        'transcription_plain': {'type': 'text'},
+                        'description_plain': {'type': 'text'},
+                        'author': {
+                            'type': 'keyword'
+                        },
+                        'keywords': {
+                            'type': 'keyword'
+                        },
+                        'file_type': {
+                            'type': 'keyword'
+                        },
+                        'id': {'type': 'text'},
+                        'external_id': {
+                            'type': 'keyword'
+                        },
+                        'arrangement_collection_name': {
+                            'type': 'keyword'
+                        },
+                        'educational_levels': {
+                            'type': 'keyword'
+                        },
+                        'lom_educational_levels': {
+                            'type': 'keyword'
+                        },
+                        'disciplines': {
+                            'type': 'keyword'
+                        },
+                        "suggest" : {
+                            "type" : "completion"
+                        },
+                    }
+                }
+            }
+        }
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
Lijkt me nog steeds een goed idee om zelf gemaakte https://docs.djangoproject.com/en/2.2/topics/testing/tools/#django.test.TestCase.setUpTestData hier aan te roepen, zodat je heel consistent per test case data kan inladen. Maar hoeft niet in deze PR.

Reply:
Dan begrijp ik het denk ik niet goed. Voel je vrij om het te verbeteren 😉

Reply:
Ok, zal morgen ochtend even PR maken